README.md

```diff
@@ -1,6 +1,28 @@
 foxsi-science
 =============
 
+INTALL:
+    
+    Include this five lines in your personal IDL_STARTUP file
+
+    COMMON FOXSI,FOXSIPKG,FOXSIDB
+    FOXSIPKG='path/to/your/foxsi-science/directory'
+    FOXSIDB='path/to/your/foxsi-data/directory/foxsidb'
+    add_path,FOXSIPKG
+    add_path,FOXSIDB
```

Comment:
We should add some guidance as to which files should be put within the foxsidb folder.  

Side note, just in case it's useful -- the add_path procedure has a keyword /expand that can be used to add all subdirectories in addition to the directory you specify.